tests/test_kldiv.py

```diff
@@ -17,13 +17,40 @@
 import numpy as np
 import pytest
 import tensorflow as tf
-from numpy.testing import assert_almost_equal
+from numpy.testing import assert_allclose, assert_almost_equal
 
-from gpflow.kullback_leiblers import gauss_kl
-from gpflow import default_float, default_jitter
+import gpflow
+from gpflow import default_float, default_jitter, Parameter
+from gpflow.inducing_variables import InducingPoints
+from gpflow.kullback_leiblers import gauss_kl, prior_kl
+from gpflow.utilities.bijectors import triangular
 
 rng = np.random.RandomState(0)
 
+# ------------------------------------------
+# Fixtures
+# ------------------------------------------
+
+Ln = 2
+Nn = 10
+Mn = 50
```

Comment:
Would the test be faster but just as useful if these were smaller ?

Reply:
(it may not be much difference for any individual test, but with lots of tests even milliseconds can start to matter)